lib/scoped_model.dart

```diff
@@ -66,7 +66,7 @@ abstract class Model extends Listenable {
         // Convert the Set to a List before executing each listener. This
         // prevents errors that can arise if a listener removes itself during
         // invocation!
-        _listeners.toList().forEach((VoidCallback listener) => listener());
+        _listeners.forEach((VoidCallback listener) => listener());
```

Comment:
Please revert this line! As the comment above it suggests, this conversion isn't done so we can loop over the set, it's done to prevent modification to the Set during iteration if one of the `listener` functions that is invoked calls `add` or `removeListener` inside.

Reply:
Ok. I'll do.